src/Uno.UI/Media/PathStreamGeometryContext.cs

```diff
@@ -133,11 +133,21 @@ public override void ArcTo(Point point, Size size, double rotationAngle, bool is
 			);
 
 #elif __MACOS__
-			bezierPath.AppendPathWithArc(center,
+//Ugly workaround. check if all vars are defined
+			if (!double.IsNaN(radius) && !double.IsNaN(startAngle) && !double.IsNaN(endAngle)) {
+
+				//Convert to degrees in a 0 =< x =< 360 deg range
+				startAngle = (startAngle * (180 / PI) + 360) % 360;
```

Comment:
We already have a helper for that:
```suggestion
				startAngle = MathEx.ToDegreeNormalized(startAngle);
```